Dockerfile

```diff
@@ -3,19 +3,19 @@
 # -v "$PWD/.terraform:/data-center-terraform/.terraform \
 # -v "$PWD/logs:/data-center-terraform/logs" \
 # -v "$PWD/config.tfvars:/data-center-terraform/config.tfvars" \
-# -it localtf ./install.sh -c config.tfvars
+# -it atlassianlabs/terraform:main ./install.sh -c config.tfvars
 #
 # Example docker command to collect k8s logs
 # docker run --env-file aws_envs \
 # -v "$PWD/k8s_logs:/data-center-terraform/k8s_logs" \
 # -v "$PWD/logs:/data-center-terraform/logs" \
-# -it localtf ./scripts/collect_k8s_logs.sh atlas-cluster-name-cluster us-east-2 k8s_logs
+# -it atlassianlabs/terraform:main ./scripts/collect_k8s_logs.sh atlas-cluster-name-cluster us-east-2 k8s_logs
 
 # In those example aws_envs should contain AWS variables needed for authorization like:
 # AWS_SECRET_ACCESS_KEY="asd123asd123"
 # AWS_ACCESS_KEY_ID="123dsa321asd"
 
-FROM ubuntu:22.04
+FROM ubuntu:18.04
```

Comment:
@bianchi2 we need some help here. We could not build an image on bamboo CI with ubuntu 22.04 due to gpg error.
@dmika1 could you please add more details?

Reply:
Do we have a log file?

Reply:
The problem is already solved. No ubuntu downgrade needed and this PR can be rejected. I'll create today PR with just modified docker image name from localtf to atlassianlabs/terraform.